docs/index.bs

```diff
@@ -3032,6 +3032,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
                   1. Wait until |e|'s [=FetchEvent/wait to respond flag=] is unset.
                   1. If |e|'s [=FetchEvent/respond-with error flag=] is set, set |handleFetchFailed| to true.
                   1. Else, set |response| to |e|'s [=FetchEvent/potential response=].
+              1. If |response| is null, |request|'s [=request/body=] is not null, |request|'s [=request/body=]'s [=Body/source=] is null, and |request|'s [=request/body=] is [=Body/unusable=], set |handleFetchFailed| to true.
```

Comment:
The second check here is redundant with checking whether it is unusable. In order for a body to be unusable it has to be non-null.

Reply:
Makes sense, but I'd keep this because the next change added a cancel step.